internal/wrappers/client.go

```diff
@@ -603,6 +592,40 @@ func request(client *http.Client, req *http.Request, responseBody bool) (*http.R
 	return nil, err
 }
 
+func handleRedirect(resp *http.Response, req *http.Request, body []byte) (*http.Request, error) {
+	redirectURL := resp.Header.Get("Location")
+	if redirectURL == "" {
+		return nil, fmt.Errorf("redirect URL not found in response")
```

Comment:
suggestion:
use constants for the error messages. we have a file application-errors.go